services/QuillLMS/engines/comprehension/test/dummy/config/environments/test.rb

```diff
@@ -13,8 +13,8 @@
   config.eager_load = false
 
   # Configure static file server for tests with Cache-Control for performance.
-  config.serve_static_files   = true
-  config.static_cache_control = 'public, max-age=3600'
+  config.public_file_server.enabled = true
```

Comment:
Just a heads up to @happythenewsad, I made some of these same changes in #7743 , so you may get conflicts when you merge (You can take the ones from `develop` when it happens).